lib.js

```diff
@@ -82,9 +79,10 @@ module.exports = (options) => {
             }
         }));
     };
-
+    console.log("Starting...this can take a couple of minutes");
```

Comment:
My idea was to be a little more verbose in the progress of the process. Of course, now it's a little faster considering it only fetches only one project, but I still think this should be presented (maybe removing the "minutes" part)

This is how the output looks on a sucessfull call (has elapsed seconds on the bottom right corner) 
![201608251432-capture](https://cloud.githubusercontent.com/assets/595815/17979573/b45469b4-6ad1-11e6-9c40-083c51987c6b.gif)

and this how it looks on a gitlab fetch failure 
![201608251440-capture](https://cloud.githubusercontent.com/assets/595815/17979626/00ae868c-6ad2-11e6-828c-0cee0fc43f17.gif)

Note that gitlab can take it's time to respond.

If you still think this has no place here, I'll remove it. no problem